src/MainMenu.cpp

```diff
@@ -311,3 +315,235 @@ ZunResult MainMenu::DrawStartMenu(void)
     return ZUN_SUCCESS;
 }
 #pragma optimize("", on)
+
+void _strcpy(char *dst, char *src)
+{
+    strcpy(dst, src);
+}
+
+#pragma optimize("s", on)
+#pragma var_order(anmVm, cur, replayFileHandle, replayFileIdx, replayData, replayFilePath, replayFileInfo, uh, uh2,    \
+                  padding)
+i32 MainMenu::ReplayHandling()
+{
+    AnmVm *anmVm;
+    i32 cur;
+    HANDLE replayFileHandle;
+    u32 replayFileIdx;
+    ReplayData *replayData;
+    char replayFilePath[32];
+    WIN32_FIND_DATA replayFileInfo;
+    u8 padding[0x20]; // idk
+
+    switch (this->gameState)
+    {
+    case STATE_REPLAY_LOAD:
+        if (this->stateTimer == 0x3c)
+        {
+            if (LoadReplayMenu(this))
+            {
+                GameErrorContextLog(&g_GameErrorContext, "japanese");
+                g_Supervisor.curState = 4;
+                return ZUN_SUCCESS;
+            }
+            else
+            {
+                replayFileIdx = 0;
+                for (cur = 0; cur < 15; cur++)
+                {
+                    sprintf(replayFilePath, "./replay/th6_%.2d.rpy", cur + 1);
+                    replayData = (ReplayData *)FileSystem::OpenPath(replayFilePath, 1);
+                    if (replayData == NULL)
+                    {
+                        continue;
+                    }
+                    if (!validateReplayData(replayData, g_LastFileSize))
+                    {
+                        // FIXME: wrong assembly
+                        memcpy(&this->replayFileData[replayFileIdx], replayData, 0x50);
+                        // HACK: i dont think it should be this way
+                        _strcpy(this->replayFilePaths[replayFileIdx], replayFilePath);
+                        sprintf(this->replayFileName[replayFileIdx], "No.%.2d", cur + 1);
+                        replayFileIdx++;
+                    }
+                    free(replayData);
+                }
+                FileSystem::CreateDirectoryInCWD("./replay");
```

Comment:
This is almost certainly a function from the C standard library, such as [`_mkdir`](https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/mkdir-wmkdir?view=msvc-170).